sites/melbourne/app/models/melbourne/event/open_graph_metadata.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Melbourne
+  class Event
+    private_constant :OpenGraphMetadata
+    class OpenGraphMetadata
+      def initialize(event)
+        @event = event
+      end
+
+      def to_meta_tags # rubocop:disable Metrics/MethodLength
```

Comment:
Same as above; seemed a bit excessive but I can change it if anyone feels strongly